pkg/commands/workflows/remove_comments.go

```diff
@@ -84,13 +104,23 @@ func (c *RemovePlanCommentsCommand) Flags() *cli.FlagSet {
 			merr = errors.Join(merr, fmt.Errorf("missing flag: pull-request-number is required"))
 		}
 
+		if len(c.flagCommentTypes) == 0 {
+			merr = errors.Join(merr, fmt.Errorf("missing flag: comment-type is required"))
+		}
+
+		for _, commentType := range c.flagCommentTypes {
+			if _, ok := commentTypePrefixes[commentType]; !ok {
```

Comment:
I think you could use sets.Subtract here, for example https://github.com/abcxyz/abc/blob/3b502883f47d960b9258e566c53c34e52676634f/templates/common/builtinvar/builtin_vars.go#L51

Reply:
nice!